requirements/main.txt

```diff
@@ -1,3 +1,4 @@
+differentiable-robot-model>=0.2.1
```

Comment:
Is this the latest version where you added the efficient computation of manipulator jacobian?

Reply:
I only added a more efficient version of computing forward kinematics for all joints. Once that PR is merged into differentiable-robot-model I will make a separate PR to adopt it on the Theseus side.

Reply:
So to explicitly answer your question, this is still the inefficient version.